.github/workflows/pr-build.yaml

```diff
@@ -26,6 +26,12 @@ jobs:
         with:
           fetch-depth: 0
 
+      # On the Mac build machines, libssl 1.1 is installed
+      # but not linked in the path expected by .NET. Fix it now.
+      - name: Link libssl on Mac OS
+        run: ln -sn /usr/local/opt/openssl/lib/libssl.1.1.dylib /usr/local/lib/libssl.1.1.dylib
+        if: matrix.os == 'macOS-latest'
```

Comment:
Can there be a condition that checks if `libssl` is installed?

Reply:
`ln` will fail with nonzero exit code if a file exists at that path; it won't overwrite without the `-f` option.

If that happens the build will fail, and we can address it then. But I don't see a need to make a more complex script now.